modin/pandas/io.py

```diff
@@ -992,6 +995,9 @@ def from_pandas(df) -> DataFrame:
     """
     from modin.core.execution.dispatching.factories.dispatcher import FactoryDispatcher
 
+    if UsePlainPandasQueryCompiler.get():
+        return ModinObjects.DataFrame(query_compiler=PlainPandasQueryCompiler(df))
```

Comment:
Maybe we should move this to BaseFactory._to_pandas?

Reply:
Maybe you mean `from_pandas`?

Reply:
moved to BaseFactory.from_pandas